install_on_raspberry_fr.md

```diff
@@ -2,21 +2,26 @@
 
 *Toutes les autres façons d’installer YunoHost sont listées **[ici](/install_fr)**.*
 
+<center>
+<img src="https://cdn.shopify.com/s/files/1/0176/3274/products/rpi2b_1024x1024.jpg" width=350>
+</center>
+
+<div class="alert alert-info" markdown="1">
+Avant d'héberger un serveur chez vous, il est recommandé de prendre connaissance des [limitations liées à votre FAI](/isp). Si votre FAI est trop contraignant, il est possible d'[utiliser un VPN pour contourner ces limitations](/vpn_advantage_fr) 
+</div>
+
 ## Prérequis
-<img src="/images/Raspberry_Pi_2_Model_B_v1.1_front_angle_new.jpg" width=350>
-<img src="/images/micro-sd-card.jpg">
-
-- Un Raspberry Pi 1, 2 ou 3
-- Une carte SD de capacité minimale de **4 Go** et une certification de vitesse **class10** sont recommandées
-- Un autre ordinateur pour parcourir ce guide et accéder à votre Raspberry Pi
-- Un écran et un clavier sont recommandés pour pouvoir contrôler votre Raspberry Pi si un problème apparaît
-- Un [fournisseur d’accès correct](/isp_fr), de préférence avec une bonne vitesse d’upload
-- Les **images YunoHost pour Raspberry Pi**, disponible ici :
+
+- Un Raspberry Pi 1, 2 ou 3 ;
+- Une carte SD *classe 10* d'au moins 8 Go (par exemple une [Transcend 300x](http://www.amazon.fr/Transcend-microSDHC-adaptateur-TS32GUSDU1E-Emballage/dp/B00CES44EO)) ;
+- Un adaptateur secteur pour la alimenter la carte ;
+- Un câble ethernet/RJ-45 pour brancher la carte à votre routeur/box internet.
```

Comment:
Hm la question c'est est-ce que c'est souhaitable d'avoir un serveur connecté en wifi ? Et est-ce que ça requiert de la configuration supplémentaire, ou est-ce que ce genre d'adaptateur marche out of the box ?